src/index.js

```diff
@@ -0,0 +1,69 @@
+
+const path = require('path');
+const fs = require('fs');
+const fetch = require('node-fetch');
+
+
+
+const inquirer = require('inquirer');
+inquirer
+  .prompt({
+   name: 'archivo',
+   message: '¿Que archivo deseas leer',
+       
+  })
+  .then(answers => {
+      let archivo = answers.archivo;
+     console.log(archivo)
+     validateMd(archivo)
+  })
+  .catch(error => {
+    if(error.isTtyError) {
+      // Prompt couldn't be rendered in the current environment
+    } else {
+      // Something else when wrong
+    }
```

Comment:
No esta bien definido el catch
```suggestion
  .catch(error => {
    if(error.isTtyError) {
      // Prompt couldn't be rendered in the current environment
    } else {
      // Something else when wrong
    }
```